docs/reference/query-dsl/geo-polygon-query.asciidoc

```diff
@@ -6,25 +6,29 @@ points. Here is an example:
 
 [source,js]
 --------------------------------------------------
+GET /_search
 {
-    "bool" : {
-        "query" : {
-            "match_all" : {}
-        },
-        "filter" : {
-            "geo_polygon" : {
-                "person.location" : {
-                    "points" : [
+    "query": {
+        "bool" : {
+            "must" : {
```

Comment:
I had the same thought when making those changes but decided to keep changes minimal and post-pone this to a later PR.

It does parse successfully without being surrounded by the bool query. Not sure if this is a left-over of the filter to query migration. @nknize @clintongormley